packages/components/app/styles/components/app-footer.scss

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// app-footer
+//
+
+// WIP: names, etc. may change
+// :root {
+//   --token-app-footer-text-color: var(--token-color-foreground-faint);
+//   --token-app-footer-border-color: var(--token-color-border-primary);
+// }
+
+// NOTE: Styles are a WIP and NOT final
+.hds-app-footer {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 24px;
+  justify-content: flex-end;
+
+  padding: 24px;
+  border-top: 1px solid var(--token-color-border-primary);
```

Comment:
this, in conjunction with the fact that now everything is inside a flex list, makes me nervous.
what if consumers want to achieve this layout/design?

![aaa](https://github.com/hashicorp/design-system/assets/686239/f6a42de1-3a05-41f9-ba3f-50ad3b5bf6e7)

(I know they can use a container with 100% width, but is not... very elegant)

Reply:
The visual design was updated to not support this so I updated the component. You want me to change it back to what it was before? @andgen404

Reply:
If a decision has been made at design level that this in not allowed, for consumers, than that's OK. I personally think that someone sooner or later will come up with a request like this, and we'll be stuck, needing to reconsider this decision and refactor the component.

Reply:
I don't know yet how to handle this. The visual design still shows everything inline for the widest view.